setup.py

```diff
@@ -38,8 +38,16 @@
       maintainer_email='mrocklin@gmail.com',
       license='BSD',
       keywords='task-scheduling parallel numpy pandas pydata',
+      classifiers=[
+          "Programming Language :: Python :: 2",
+          "Programming Language :: Python :: 2.7",
+          "Programming Language :: Python :: 3",
+          "Programming Language :: Python :: 3.5",
+          "Programming Language :: Python :: 3.6",
```

Comment:
3.7 is also supported, so much as our dependencies support it.

Reply:
Good to hear! I've added 3.7.